### DISCUSSION TITLE
Vanilla JS example

### DISCUSSION THREAD

**Discussion by kuipou** [supabase/supabase#12332] [Feature Requests]
It would be a good idea to have example without any frameworks for example vanilla js or an express server

**Comment by Anikets08** [comment]
I can do this stuff, let me know. Thanks

**Comment by kiwicopple** [comment]
we'd love some help with this @Anikets08 . 

For now I'll move this to a Discussion so we can use issues for bugs that need immediate attention

**Comment by Anikets08** [comment]
on it, thanks confirming @kiwicopple

**Comment by kuipou** [comment]
any movement?
